### PR TITLE
Updated Readme to include installation Instructions for Quilt

### PR DESCRIPTION
**Pull request by Redfan2**
I updated the Text and the Links in the Readme to fit for Quilt since the old Readme was entirely written for the old Fabric Version. I also edited the Requiem Logo to include a light version which only shows in Dark Mode since the Dark logo is poorly readable in dark Mode. Please give your Feedback on this Featue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,45 @@
 # Requiem
 
-![](https://raw.githubusercontent.com/Ladysnake/Requiem/master/requiem-logo-200x125.png)
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="requiem-logo-200x125_light.png" width="250">
+        <img src="requiem-logo-200x125.png" width="250">
+    </picture>
```
Comment (Pyrofab) [inline README.md]: Never thought of doing that, that's really cool ! The light version seems hella crispy though.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,45 @@
...
+    </picture> 
+</p>
 
 [![Download](https://curse.nikky.moe/api/img/265729?logo)](https://curse.nikky.moe/api/url/265729) ![](https://jitpack.io/v/ladysnake/dissolution.svg) ![](https://img.shields.io/github/issues/ladysnake/requiem.svg)
 
+<p>
+
+[<img alt="Quilt Logo" width="400px" src="https://raw.githubusercontent.com/QuiltMC/art/master/contributed-art/doctor4t/quilt-exclusive-banner.png?raw=true" />](https://quiltmc.org/en/about/faq/#usage-questions)
+
+</p>
```
Comment (Pyrofab) [inline README.md]:
```suggestion
```

I don't think this banner is necessary for a GitHub readme actually, curseforge and modrinth should be enough

File: README.md [hunk 3]
```diff
@@ -1,32 +1,45 @@
...
+
+
 Requiem is a Minecraft (Java Edition) mod aiming at removing the continuity break resulting from death,
 as well as adding new mechanics centered around soul dissociation and body possession.
 
-More information is available on the Ladysnake Website: https://ladysnake.github.io/wiki/requiem.
+More information is available on the Ladysnake Website: <https://ladysnake.github.io/wiki/requiem>.
```
Comment (Pyrofab) [inline README.md]: Does this do anything on GitHub ? In any case, might as well update the url to ladysnake.org

File: README.md [hunk 4]
```diff
@@ -1,32 +1,45 @@
...
+
+*   If Quilt is installed on your Minecraft Java copy, you can download Requiem from [CurseForge](https://minecraft.curseforge.com/projects/requiem), [Github Releases](https://github.com/Ladysnake/Requiem/releases), or [Modrinth](https://modrinth.com/mod/requiem). After that, copy the jar file obtained, as well as the Quilted Fabric API jar file in the mods folder located in you Minecraft installation or your Server's Mods folder. Please make sure to have the latest version, as we will not accept issues regarding older versions.
+
+*   Once the previous steps have been completed, boot up your game, and  upon your first death, Requiem will asks you whether you'd like to  enable the modified death system or not through an interactive dialogue. Be careful, because after choosing, the dialogue won't be prompted a second time, and you will have to use a special item to change that.
```
Comment (Pyrofab) [inline README.md]: Why change the list style ?

**Review by Pyrofab — COMMENTED** [review]
Neat, had just a couple questions before merging

File: README.md (hunk 1 above)
Comment (Redfan2) [reply README.md]: Maybe a grey version with double the resolution would fit better. Curious, do you like the idea of different Logos based on the Color scheme?

File: README.md (hunk 3 above)
Comment (Redfan2) [reply README.md]: To be real, I didn't even notice these Brackets. Seems like some auto-formatting magic by Milkdown, the Markdown Editor I used.

File: README.md (hunk 4 above)
Comment (Redfan2) [reply README.md]: Same as above, didn't actively change this. Suppose that Milkdown changed it, feel free to undo it.

File: README.md (hunk 1 above)
Comment (Pyrofab) [reply README.md]: I quite like the variants, just need to fix the white version's resolution IMO - what tool did you use to swap the colors ?

Comment (Redfan2) [reply README.md]: MS Paint 3D

File: README.md (hunk 3 above)
Comment (Pyrofab) [reply README.md]: Ah I see. It's a bit annoying to have those unnecessary changes though - have you tried using [HackMD](https://hackmd.io) instead ?

Comment (Redfan2) [reply README.md]: I haven't. Thanks for suggesting it

**Pull request merged**